### ISSUE TITLE
ImportError: No module named 'kolibri.tasks'

### ISSUE THREAD

**Issue by mrpau-eduard** [learningequality/kolibri#419]
## Summary

The `kolibri.tasks ` module was not found during creating the `sdist ` source distribution.

## System information

 - Version: `master`

## Traceback or relevant snippet from server.log

```
(kolibritest) echo-air:dist arceduardvincent$ kolibri start
Traceback (most recent call last):
  File "/Users/arceduardvincent/.virtualenvs/kolibritest/bin/kolibri", line 11, in <module>
    sys.exit(main())
  File "/Users/arceduardvincent/.virtualenvs/kolibritest/lib/python3.5/site-packages/kolibri/utils/cli.py", line 232, in main
    django.setup()
  File "/Users/arceduardvincent/.virtualenvs/kolibritest/lib/python3.5/site-packages/kolibri/dist/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/arceduardvincent/.virtualenvs/kolibritest/lib/python3.5/site-packages/kolibri/dist/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/Users/arceduardvincent/.virtualenvs/kolibritest/lib/python3.5/site-packages/kolibri/dist/django/apps/config.py", line 116, in create
    mod = import_module(mod_path)
  File "/Users/arceduardvincent/.virtualenvs/kolibritest/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 944, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 956, in _find_and_load_unlocked
ImportError: No module named 'kolibri.tasks'
(kolibritest) echo-air:dist arceduardvincent$
```

## How to reproduce
1. Used [virtualenv](http://virtualenvwrapper.readthedocs.io/en/latest/index.html) then navigate to `~/kolibri`.
1. Run `python setup.py sdist --static --format=zip`. When it is finished navigate to  `~/kolibri/dist`.
1. Install the `zip` file via pip.
1. Run `kolibri start`. Notice that the local server fails to start.

**Comment by aronasorman** [comment]
Will look into this today.

**Comment by aronasorman** [comment]
Fixed in #420.
